package/gluon-mesh-vpn-wireguard/files/usr/bin/nc_timeout

```diff
@@ -0,0 +1,21 @@
+#!/bin/sh
```

Comment:
this file is not wireguard-specific. So I would suggest to move it to another PR

Reply:
#1577 is there for this :)

Reply:
the file is exclusively used by wireguard. Let's move it when we have another use case.